### DISCUSSION TITLE
Beta.character.ai api

### DISCUSSION THREAD

**Discussion by Yorddyyorddy** [acheong08/EdgeGPT#457] [General]
Pls someone make an api for beta.character.ai  or suggest me if there, made without the use of selenium and playwright coz they can't used in projects or in replit ide

Also if you know how to use selenium or playwright without pc (means in android,or py projects)  you can share me too pls

**Comment by acheong08** [comment]
I wrote a tutorial on how to make your own https://gist.github.com/acheong08/be0b43bb89f9f4797079e5948c48738c

**Reply by acheong08** [reply]
It can be done without browser automation via tls spoofing
